src/jit/lsraxarch.cpp

```diff
@@ -1177,26 +1252,16 @@ void LinearScan::BuildCall(GenTreeCall* call)
 // Return Value:
 //    None.
 //
-void LinearScan::BuildBlockStore(GenTreeBlk* blkNode)
+int LinearScan::BuildBlockStore(GenTreeBlk* blkNode)
 {
-    TreeNodeInfo* info    = currentNodeInfo;
-    GenTree*      dstAddr = blkNode->Addr();
-    unsigned      size    = blkNode->gtBlkSize;
-    GenTree*      source  = blkNode->Data();
+    GenTree*     dstAddr          = blkNode->Addr();
+    unsigned     size             = blkNode->gtBlkSize;
+    GenTree*     source           = blkNode->Data();
+    int          srcCount         = 0;
+    int          internalCount    = 0;
+    const int    MaxInternalCount = 4;
+    RefPosition* internalDefs[MaxInternalCount];
```

Comment:
Not used it seems. Same for internalCount above. Strange that they have the same name as 2 LinearScan data members.

Reply:
Yes; I had originally added these to each method needing internal temps, but it was quite a bit messier than handling it pseudo-globally. I'll delete these.